rfc/2022/2022-07-14_Platform_Security_001_Service_Account_Guidance.md

```diff
@@ -0,0 +1,112 @@
+# RFC: Service Account Guidance
+
+<!--
+The title is what you want comments on. Use the active voice in a future tense.
+Example:
+    - The website will be built using the XZY framework
+-->
+
+* Comment Deadline: `2022-08-14`
+* Team Crew and Name: Platform Security
+* Authors:
+  * [Troy Mosher](https://github.com/troymosher)
+  
+ * [Original RFC Pull Request](https://github.com/department-of-veterans-affairs/va.gov-platform-architecture/pull/35)
+
+## Background
+
+Numerous service accounts exist throughout the Platform, and there is currently no central tracking of them, nor consistent guidance on how they are created, approved, managed, and maintained. 
+
+## Motivation
+
+The proposed Service Account Guidance would provide guidance on how service accounts should be requested, approved, managed and maintained.
+
+## Design
+
+The proposed [Service Account Guidance](https://vfs.atlassian.net/wiki/spaces/PSEC/pages/2222588013/Service+Account+Guidance) is a document in the PSEC Confluence page.
+
+Content below:
+
+#### What is a Service Account?
+A service account is a special type of shared/group privileged account used to execute applications, services, or other processes.  Service accounts are used to run automated or persistent processes and are intended to be used by applications, not interactively by people.
+
+#### What unique risks do Service Accounts pose?
+Service accounts are not tied back to a unique individual.   This makes tracing back actions performed by a service account to a specific individual a challenge. 
+
+Service account authenticators are often shared or known amongst a team.  This necessitates rotation of authenticators when a member of that team leaves the project.
+
+Service account authenticators are often stored within applications or code for use, and rotation of that authenticator can cause outage or downtime if not updated in all the applications, services, or other processes that rely upon its use.
+
+#### Platform Security Guidance for Service Accounts
+
+*Assigned Account Manager*
+
+Every service account should have an assigned Account Manager.  The Account Manager is a specific human individual with the overall responsibility for the service account but can delegate use of the account.  The Account Manager can be a team lead or member of the team that necessitates the use of the service account.  In the case of an Account Manager leaving, a replacement Account Manager from the team must be designated.
```

Comment:
Who determines the account manager? Who designates replacement account managers?

Reply:
In most cases it would probably be the team lead for the team that requires the service account.

Reply:
resolved in https://github.com/department-of-veterans-affairs/va.gov-platform-architecture/pull/35/commits/89c1862893ee9df4b3477a78e5c5fc1e19e834ba